catalog/ui/catalog-ui-search/src/main/webapp/component/query-title/query-title.hbs

```diff
@@ -11,7 +11,7 @@
  *
  **/
  --}}
-<input class="query-title" placeholder="Search Title" value="{{this.title}}" data-help="Displays the title given to the query."/>
+<input class="query-title" placeholder="Search Name" value="{{this.title}}" data-help="Displays the title given to the query."/>
```

Comment:
We should update the data-help as well since it says title.

Reply:
Fixed